### ISSUE TITLE
Better sort and filter features

### ISSUE THREAD

**Issue by reissmann** [ddserver/ddserver#32]
All lists should be sortable by column. 

Also filters (at least for admins) would be nice to see only hostnames in a specific domain or for a specific user.

Depends on #9

**Comment by reissmann** [comment]
Sort function done, filters still missing. Would be nice to have filters like:
 - All users with 0 hostnames
 - All users with lastlogin > 1 year
 - All hostnames not updated for 1 year
 - ...
